includes/classes/PullListTable.php

```diff
@@ -222,35 +225,37 @@ public function column_date( $post ) {
 	/**
 	 * Output standard table columns (not name)
 	 *
-	 * @param  array  $item Item to output.
-	 * @param  string $column_name Column name.
+	 * @param  array|\WP_Post $item Item to output.
+	 * @param  string         $column_name Column name.
+	 *
+	 * @return string Url, post title, or empty string.
 	 * @since  0.8
 	 */
 	public function column_default( $item, $column_name ) {
 		switch ( $column_name ) {
 			case 'name':
-				return $item['post_title'];
-				break;
+				return $item['post_title']; // return is just as good as break
 			case 'url':
 				$url = get_post_meta( $item->ID, 'dt_external_connection_url', true );
 
 				if ( empty( $url ) ) {
 					$url = esc_html__( 'None', 'distributor' );
 				}
 
-				return $url;
-				break;
+				return $url; // no need to break, return will do.
```

Comment:
Change looks good, I'd remove the comment though, don't think it adds much here.